request/request.go

```diff
@@ -94,6 +109,24 @@ func (c *RequestClient) makeRequestAndParseResult(req *http.Request, result any)
 	return res, nil
 }
 
+func (c *RequestClient) makePlainTextRequestAndParseResult(req *http.Request) (*http.Response, error) {
```

Comment:
Doesn't seem like this is doing "plain text". I don't see any mention of `Content-Type` (the response content type that is) or any parsing at all. Seems like a more apt name would be `makeRequestAndIgnoreResult`. Or, if you really are expecting to deal with just text, at least check `Content-Type` of the response to ensure that it starts with `text/` (at least then you'll be safe to stringify the body). Realistically though our API should never be returning `text/plain` in our responses.

Reply:
If we remove line 125 `logger.Info(string(payload))` (which is no longer necessary if you follow my suggestion in https://github.com/amp-labs/cli/pull/51/files#r1383786181) , then this method can simply be `makeRequest`, and we don't have to be concerned with response type.